external-deps/qtconsole/docs/source/conf.py

```diff
@@ -50,7 +50,7 @@
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = {".rst": "restructuredtext"}
```

Comment:
Please revert this change. This is a copy of the Qtconsole repo that's managed by [git subrepo](https://github.com/ingydotnet/git-subrepo), so it can't be touched directly.

Reply:
Done.